src/helpers/AvatarHelper.ts

```diff
@@ -27,13 +27,13 @@ export function getColorById(id: string, avatarColors = getAvatarColors()) {
   return avatarColors[colorIndex];
 }
 
-export function getInitials(name: string) {
+export function getInitials(name?: string, limit = 2) {
```

Comment:
Consider adding proper unit test for the new limit argument.

Reply:
Added